### PR TITLE
Readme draft

### LINKED ISSUE #3: Update the Readme

### PR DESCRIPTION
**Pull request by amuradyan**
@oshn @Karlengevorgyan91 Check this out. Not neon but with _dancing_ rainbow colors.

----
See also #5 and #3

### CODE REVIEW

**Comment by amuradyan** [comment]
I used [this](https://en.bloggif.com/text) resource to generate the text.

**Comment by Karlengevorgyan91** [comment]
I think it's better. not neon  but very colorful

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<img src="./images/Welcome.gif" alt="Welcome to SuperHotWinXRoyal777" width="100%"/>
+
+![Slot Machine Hall](./images/slot_machine.jpg)
+
+This is a [slot machine](https://en.wikipedia.org/wiki/Slot_machine) engine in Java.
+
+----------
+
+## NOTES
+
```
Comment (oshn) [inline Readme.md:12]:
```suggestion

### Credits

```

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# SuperHotWinXRoyal777 🎰
+
+<img src="./images/Welcome.gif" alt="Welcome to SuperHotWinXRoyal777" width="100%"/>
```
Comment (oshn) [inline Readme.md:3]: Not bad, you're on a right track. 

I'd take some time to experiment around further, maybe after this PR is merged.

**Review by oshn — DISMISSED** [review]

File: Readme.md (hunk 2 above)
Comment (amuradyan) [reply Readme.md:3]: Feel free to feel free.

Comment (oshn) [reply Readme.md:3]: Question: you put `<img ..` here instead of markdown image so you could set width to 100%?

Comment (amuradyan) [reply Readme.md:3]: Yes

**Comment by amuradyan** [comment]
@oshn Do you have any suggestions we can discuss now? If not, I'd like to merge this, so we have a readme.

**Review by oshn — APPROVED** [review]
Ship it!

**Pull request merged**